Content.Server/_Harmony/Explosion/Components/TriggerOnUiOpenComponent.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Content.Server.Explosion.Components;
+
+/// <summary>
+/// Triggers when StorageComponent is opened
+/// </summary>
+[RegisterComponent]
+public sealed partial class TriggerOnUiOpenComponent : Component { }
```

Comment:
Small nitpick you don't have to do this but I think empty classes are slightly cleaner when ended by a semicolon
```suggestion
public sealed partial class TriggerOnUiOpenComponent : Component;
```